easy_my_coop_dividend/__manifest__.py

```diff
@@ -33,8 +33,8 @@
 
     'author': 'Houssine BAKKALI, <houssine@coopiteasy.be>',
     'license': 'AGPL-3',
-    'version': '9.0.1.0',
-    'website': "www.coopiteasy.be",
+    'version': '9.0.1.0.0',
```

Comment:
@remytms should this version start with `12`?

Reply:
yes it should.

Reply:
Done.